### PR TITLE
364 Toggling password also changes icon now

### LINKED ISSUE #364: [364] Toggle password icon on login/register screen
**Bug display and/or description**
The password icon needs to toggle when you register or login.

**Reproduce**
- [ ] Step 1
- [ ] Step 2
- [ ] Step 3
- [ ] Step ...

**Branch**
364-hotfix-toggle-password

### PR DESCRIPTION
**Pull request by sribadal**
closes #364 

<br />

 **What is it supposed to do**  
Whenever you want to see your password you can click the eye icon. This eye icon needs to toggle to a different icon whenever clicked!

Before clicked:
image

After clicked:
image

<br />

 **How can we test your branch**  
1. Visit login page, register page
2. Type in password
3. Click on the eye icon

you should see the icon toggling to a different one.
<br />

 **Additional notes**  
Things to keep in mind regarding the PR that don't fit in the previous two headings. 

<br />

**1. General checklist**
- [x] a. Tested happy flow.
- [x] b. Tested unhappy flow.
- [x] c. No unexpected exceptions.
- [x] d. No code with high complexity. (Big O)
- [x] e. EOF newline.
- [x] f. No linter warnings or errors.

<br />

**2. If relevant, front-end checklist**
- [x] a. Responsive. (Desktop all the way to mobile 320px)
- [x] b. Using SCSS variables where relevant.
- [x] c. Relevant class names
- [x] d. No modified global SCSS properties, except if necessary and fully tested the impact.
- [x] e. Checked [caniuse](https://caniuse.com) for more modern CSS properties.

<br />

**3. If relevant, back-end checklist**
- [ ] a. No unhandled promise rejections.
- [ ] b. Seeders still functional.
- [ ] c. No primary-key violations.
- [ ] d. Normalization, at least [3NF](https://en.wikipedia.org/wiki/Database_normalization).
- [ ] e. Used guards (middleware) where necessary and tested if route is guarded.

<br />

**4. If relevant, test these browsers**
- [x] a. Chrome
- [x] b. Firefox
- [x] c. Opera
- [ ] d. Edge
- [ ] e. Safari

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.

### CODE REVIEW

File: frontend/src/components/formInput/index.js [hunk 1]
```diff
@@ -97,11 +97,17 @@ class FormInput extends Component {
...
           />
           {isPassword && (
-            <Icon
-              className={ `${styles.icon} ${styles.iconPassword}` }
-              iconName={ 'Eye' }
-              onClick={ this.togglePasswordVisible }
-            />
+            passwordVisible
+              ? <Icon
+                className={ `${styles.icon} ${styles.iconPassword}` }
```
Comment (jerohero) [inline frontend/src/components/formInput/index.js]: Spacing
```suggestion
                className={ `${ styles.icon } ${ styles.iconPassword }` }
```

File: frontend/src/components/formInput/index.js [hunk 2]
```diff
@@ -97,11 +97,17 @@ class FormInput extends Component {
...
+                iconName={ 'EyeSlash' }
+                onClick={ this.togglePasswordVisible }
+              />
+              : <Icon
+                className={ `${styles.icon} ${styles.iconPassword}` }
```
Comment (jerohero) [inline frontend/src/components/formInput/index.js]: Spacing
```suggestion
                className={ `${ styles.icon } ${ styles.iconPassword }` }
```

**Review by jerohero — CHANGES_REQUESTED** [review]

**Review by AjobK — CHANGES_REQUESTED** [review]
Why not just toggle the iconName based on passwordVisible? Instead of switching between entire components

File: frontend/src/components/formInput/index.js (hunk 2 above)
Comment (sribadal) [reply frontend/src/components/formInput/index.js]: Heb dit veranderd!

File: frontend/src/components/formInput/index.js (hunk 1 above)
Comment (sribadal) [reply frontend/src/components/formInput/index.js]: Is veranderd!

**Review by AjobK — APPROVED** [review]
Muchhhhhh better 😄 

Only 2 lines too haha

**Review by danielordaan — APPROVED** [review]
LGTM

**Pull request merged**